content/guides/references/troubleshooting.md

```diff
@@ -633,6 +633,17 @@ official fix faster.
 
 ### Edit the installed Cypress code
 
+<Alert type="warning">
+
+In Cypress 11.0, Cypress uses
+[v8 snapshots](https://v8.dev/blog/custom-startup-snapshots) to improve startup
+time. As a result, much of the Cypress code is contained in a single
+unmodifiable snapshot file. Thus, editing the Cypress code in the binary
+directly as described below may or may not work depending on where and how that
+code is executed.
+
+</Alert>
+
 The installed application comes with the fully transpiled, unobfuscated
 JavaScript source code that you can hack on. You might want to directly modify
 the installed app code to:
```

Comment:
With V8 snapshots, "fully transpiled, unobfuscated JavaScript source code" isn't true, right? I think we should remove that language and anything below that doesn't apply with snapshots. The docs only cover the current version of Cypress, so we should target this towards the `release-11.0` branch and the verbiage should be more clear about what can and can't be modified by users.

Reply:
Ah gotcha. Yeah I thought that we potentially tried to cover multiple versions, but I can update this to be specific to 11.0

Reply:
Good call. Thinking about this more, I think this section will cause more confusion than any benefit with leaving it. I decided to remove it instead:
[`dfbab20` (#4825)](https://github.com/cypress-io/cypress-documentation/pull/4825/commits/dfbab207630f5c6934cde44ce6f943a63a91ddca)